spyder/api/plugins.py

```diff
@@ -226,6 +227,20 @@ def refresh_actions(self):
         self.plugin_actions = self.get_plugin_actions() + additional_actions
         add_actions(self.options_menu, self.plugin_actions)
 
+        # Show icons in Mac plugin menus
```

Comment:
Since this method is already hooked to the about to show signal we can add the end something like this:
```python
        if sys.platform == 'darwin':
            self.verify_menu_actions(True)
```